Simplenote/Classes/Simperium+Simplenote.swift

```diff
@@ -5,13 +5,34 @@ import Foundation
 //
 extension Simperium {
 
-    /// Notes Bucket
+    /// Bucket: Account
+    /// - Note: Since it's **dynamic** (InMemory JSON Storage), we don't really have an Account class
```

Comment:
Wondering how Simperium decides which bucket is dynamic and which is not? Because initialization of Account bucket looks the same as other buckets 🤔

Reply:
Super super simple. This is not new, but... behavior I had no idea was there:

- If an Entity is in CoreData, it's not dynamic
- If a SPManagedObject subclass cannot be found, **then** It's dynamic!

🔥

Reply:
I'll update the comment for documentation purposes, thank you!!